fix change vusInitalized example (#1337)
grafana/k6-docs: src/data/markdown/docs/02 javascript api/06 k6-execution.md

| iterationsInterrupted  | integer | The number of prematurely interrupted iterations in the current instance. |
| iterationsCompleted    | integer | The number of completed iterations in the current instance. |
| vusActive              | integer | The number of active VUs. |

> | vusInitialized         | integer | The number of VUs which finished setup and are ready or running. If more VUs are required, more will be Initialized. |

Reviewer: ```suggestion
| vusInitialized         | integer | The number of currently initialized VUs. |
```

| currentTestRunDuration | float   | The time passed from the start of the current test run in milliseconds. |

> 

Reviewer: ```suggestion

`vusInitialized` vs `vusActive`

The `vusInitialized` and `vusActive` might seem similar, but they are different properties and their values might vary during a test run. For example, if you have a scenario with 10 VUs and a ramp-up duration of 10 seconds, then at the beginning of the test, `vusInitialized` will be 10 and `vusActive` will be 0. After 5 seconds, `vusActive` will be 5 and `vusInitialized` will still be 10. After 10 seconds, both `vusActive and `vusInitialized` will be 10.

This is a simple example, but their values can also vary depending on what type of scenario you're using, and your test script.

```
